### PR TITLE
ticdc: Include local:// and s3:// sink support

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### What is changed, added or deleted? (Required)

- Add documentation about the local:// and s3:// sinks in TiCDC
- Add documentation about br restore cdclog

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5520
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -238,6 +238,26 @@ URI 中可配置的的参数如下：
...
+
+下面的命令会创建一个 changefeed，将 cdclog 文件本地写入到 `/data/cdc/log` 目录。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="local:///data/cdclog" --config changefeed.toml
+```
+
+The following command creates a changefeed that will write cdclog files to an external S3 storage in the `logbucket` bucket with a subdirectory of `test`. The endpoint is set in the URI, which is needed if you are using an S3-compatible storage other than Amazon S3.
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:253]: This paragraph is not translated.

Comment (Liuxiaozhen12) [reply ticdc/manage-ticdc.md]: Done~ But I am not sure about the `which` in this sentence `The endpoint is set in the URI, which is needed if you are using an S3-compatible storage other than Amazon S3`, does it mean the endpoint or the whole setting operation? PTAL, thanks!

File: br/use-br-command-line-tool.md [hunk 2]
```diff
@@ -220,6 +220,28 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
 
 示例备份的增量数据记录 `(LAST_BACKUP_TS, current PD timestamp]` 之间的数据变更，以及这段时间内的 DDL。在恢复的时候，BR 会先把所有 DDL 恢复，而后才会恢复数据。 
 
+### 时间点恢复（实验性功能）
+
+时间点恢复（PITR）可以将数据恢复至你选择的任意时间点。
+
+示例场景：每天进行一次全量备份，每 6 小时进行一次增量备份，然后使用 TiCDC 进行 PITR。假设某天 00:00 进行全量备份，06:00 进行第一次增量备份。如果要将数据库恢复到 07:16 的状态，可以先恢复全量备份 (00:00) 和增量备份(06:00)，然后恢复 TiCDC 日志，填入 06:00 和 07:16 之间的间隔。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:227]:
```suggestion
示例场景：每天进行一次全量备份，每 6 小时进行一次增量备份，然后使用 TiCDC 进行 PITR。假设某天 00:00 进行了全量备份，06:00 进行第一次增量备份。如果要将数据库恢复到 07:16 的状态，可以先恢复全量备份数据（备份于 00:00）和增量备份数据（备份于 06:00），然后恢复 TiCDC 日志，填补 06:00 和 07:16 之间的数据。
```

File: br/use-br-command-line-tool.md [hunk 3]
```diff
@@ -220,6 +220,28 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
...
+
+要执行 PITR，您可以采用以下步骤：
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:229]:
```suggestion
要执行 PITR，你可以采用以下步骤：
```

File: br/use-br-command-line-tool.md [hunk 4]
```diff
@@ -220,6 +220,28 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
...
+2. （可选）恢复增量备份。
+3. 使用 `br restore cdclog` 恢复上次增量备份后发生的事务。完整的执行命令如下：
+
+    ```shell
+    br restore cdclog --storage local:///data/cdclog --start-ts $START_TS --end-ts $END_TS
+    ```
+
+    上述命令中：
+
+    - `local:///data/cdclog` 是 TiCDC 日志的位置。该位置可能位于本地文件系统或外部存储（如 S3）上。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:241]:
```suggestion
    - `local:///data/cdclog` 是 TiCDC 日志的路径。该路径可能位于本地文件系统或外部存储（如 S3）。
```

File: br/use-br-command-line-tool.md [hunk 5]
```diff
@@ -220,6 +220,28 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
...
+    - `$START_TS` 是上次恢复备份（全量备份或增量备份）的结束位置。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:242]:
```suggestion
    - `$START_TS` 是上次恢复备份数据（全量备份或增量备份）的结束位置。
```

File: br/use-br-command-line-tool.md [hunk 6]
```diff
@@ -220,6 +220,28 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
...
+    - `$END_TS` 是你要恢复数据的点。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:243]:
```suggestion
    - `$END_TS` 是本次数据恢复的结束位置。
```

File: ticdc/manage-ticdc.md [hunk 7]
```diff
@@ -238,6 +238,26 @@ URI 中可配置的的参数如下：
 
 更多关于 Pulsar 的参数解释，参见 [pulsar-client-go ClientOptions 文档](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ClientOptions) 和 [pulsar-client-go ProducerOptions 文档](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ProducerOptions)
 
+#### Sink URI 配置 cdclog
+
+`cdclog` 文件（TiCDC 在本地文件系统或 Amazon S3 兼容存储上写入的文件）可以与备份和恢复 (BR) 一起使用以提供时间点 (PITR) 恢复。更多详细信息，请参阅[时间点恢复（实验性功能）](/br/use-br-command-line-tool.md#时间点恢复实验性功能)。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:243]:
```suggestion
你可以将 `cdclog` 文件（TiCDC 在本地文件系统或 Amazon S3 兼容存储上写入的文件）与 Backup & Restore (BR) 工具一起使用，来进行时间点恢复 (PITR)。更多详细信息，请参阅[时间点恢复（实验性功能）](/br/use-br-command-line-tool.md#时间点恢复实验性功能)。
```

File: ticdc/manage-ticdc.md [hunk 8]
```diff
@@ -238,6 +238,26 @@ URI 中可配置的的参数如下：
...
+
+下面的命令会创建一个 changefeed，将 cdclog 文件本地写入到 `/data/cdc/log` 目录。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:245]:
```suggestion
以下命令会创建一个 changefeed，将 `cdclog` 文件写入到 `/data/cdc/log` 本地目录。
```

File: ticdc/manage-ticdc.md [hunk 9]
```diff
@@ -238,6 +238,26 @@ URI 中可配置的的参数如下：
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="local:///data/cdclog" --config changefeed.toml
+```
+
+下面的命令会创建一个 changefeed，将 cdclog 文件写入到 `logbucket` 桶中的外部 S3 存储，子目录为 `test`。端点在 URI 中设置，如果你使用的是与 S3 兼容的存储而不是 Aamazons S3，则需要 URI。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:253]:
```suggestion
以下命令会创建一个 changefeed，将 `cdclog` 文件写入到 `logbucket` 桶中的外部 S3 存储，子目录为 `test`。如果你使用的是 Amazon S3 以外的 S3 兼容的存储，则需要在 URI 中设置端点。
```

File: ticdc/ticdc-overview.md [hunk 10]
```diff
@@ -40,6 +40,8 @@ TiCDC 的系统架构如下图所示：
 
 - MySQL 协议兼容的数据库，提供最终一致性支持。
 - 以 TiCDC Open Protocol 输出到 Kafka，可实现行级别有序、最终一致性或严格事务一致性三种一致性保证。
+- `cdclog` （实验性功能）: 在本地文件系统或 Amazon S3 兼容存储上写入的文件。
```
Comment (TomShawn) [inline ticdc/ticdc-overview.md:43]:
```suggestion
- `cdclog`（实验性功能）: TiCDC 在本地文件系统或 Amazon S3 兼容存储上的写入文件。
```

File: ticdc/ticdc-overview.md [hunk 11]
```diff
@@ -40,6 +40,8 @@ TiCDC 的系统架构如下图所示：
...
+- Apache Pulsar （实验性功能）
```
Comment (TomShawn) [inline ticdc/ticdc-overview.md:44]:
```suggestion
- Apache Pulsar（实验性功能）
```

**Comment by TomShawn** [comment]
The PiTR content should not be added in docs. According to https://github.com/pingcap/docs/pull/5879.

**Pull request closed without merging**